workers/unity/Packages/io.improbable.gdk.core/Subscriptions/Subscription.cs

```diff
@@ -24,8 +24,11 @@ public interface ISubscription
     public class Subscription<T> : ISubscription
     {
         // Want a custom event thing that gives an add and invoke option
```

Comment:
This comment seems to belong somewhere else

Reply:
removing comment